### ISSUE TITLE
Connect closed

### ISSUE THREAD

**Issue by xeruf** [joshbalfour/monstercat-connect-desktop-unofficial#4]
Please state that connect is closed and thus your app does not work anymore...

**Comment by joshbalfour** [comment]
@Xerus2000 yeah, don't worry - I'm working on something a lot bigger/better which should be launching in the next month or two, I'll try and remember to reply here when it's done 😄

**Comment by xeruf** [comment]
"a lot bigger/better"
these words also fit for the monstercat application I'm writing, based on the MCatalog and new expanding for Monstercat Song Downloading & Tagging. But currently I don't receive session cookies so I don't get downloading working :c 
My Monstercat Gold is running out in 3 weeks tho and I want to download all da monstercat shit till then, but not by hand ^^
